src/index.js

```diff
@@ -120,15 +120,19 @@ const GravityFormForm = ({
   if (wasSuccessfullySubmitted) {
     const confirmation = confirmations?.find((el) => el.isDefault);
 
-    return (
-      <div className="gform_confirmation_wrapper">
-        <div
-          className="gform_confirmation_message"
-          /* eslint-disable react/no-danger */
-          dangerouslySetInnerHTML={{ __html: confirmation?.message }}
-        />
-      </div>
-    );
+    if(confirmation.isDefault && confirmation.type !== 'text') {
```

Comment:
Small discussion point - thoughts on having this only for 'page'. As a GF admin adding a confirmation 'redirect' may want to direct the user to an external URL. In this casing using `window.location.href` may be required.